pkg/agent/flowexporter/exporter/exporter_test.go

```diff
@@ -53,32 +54,39 @@ func TestFlowExporter_sendTemplateRecord(t *testing.T) {
 	// Only the element name is needed, other arguments have dummy values.
 	elemList := make([]*ipfixentities.InfoElement, 0)
 	for _, ie := range IANAInfoElements {
-		elemList = append(elemList, ipfixentities.NewInfoElement(ie, 0, 0, 0, 0))
+		element, err := ipfixregistry.GetInfoElement(ie, ipfixregistry.IANAEnterpriseID)
+		if err != nil {
+			t.Errorf("Information Element %s in IANA registry does not exist.", ie)
+		}
+		elemList = append(elemList, element)
 	}
 	for _, ie := range IANAReverseInfoElements {
-		elemList = append(elemList, ipfixentities.NewInfoElement(ie, 0, 0, antreaEnterpriseRegistry, 0))
+		element, err := ipfixregistry.GetInfoElement(ie, ipfixregistry.ReverseEnterpriseID)
```

Comment:
If mock expect calls are added below, then you do not have to add the actual elements and stick with dummy elements.